src/test/ui/unsafe/rfc-2585-unsafe_op_in_unsafe_fn.stderr

```diff
@@ -74,12 +74,18 @@ LL | #![deny(unused_unsafe)]
 error: unnecessary `unsafe` block
   --> $DIR/rfc-2585-unsafe_op_in_unsafe_fn.rs:51:5
    |
+LL | unsafe fn allow_level() {
+   | ----------------------- because it's nested under this `unsafe` fn
+...
 LL |     unsafe { unsf() }
    |     ^^^^^^ unnecessary `unsafe` block
```

Comment:
FWIW, this error is kind of confusing because the "because" comes *before* the thing it is justifying. I guess there's also the error 'title' thing "unnecessary `unsafe` block", so it's probably okay, but just reading the spans things are in the wrong order... not sure how we usually handle situations where code order and "justification order" disagree.

Reply:
This is how the warning is emitted today (without `unsafe_block_in_unsafe_fn`). I guess any improvement here should be made in another PR